mftf/2.3/reporting.md

```diff
@@ -0,0 +1,368 @@
+### Test result report
+
+After the MFTF completed test execution, it generates general report about test results along with detailed information about each fail.
+
+```terminal
+--------------------------------------------------------------------------------
+DEPRECATION: Calling the "Symfony\Component\BrowserKit\Client::getInternalResponse()" method before the "request()" one is deprecated since Symfony 4.1 and will throw an exception in 5.0. /Users/.../magento2ce/vendor/symfony/browser-kit/Client.php:208
+
+Time: 52.43 seconds, Memory: 16.00MB
+
+There was 1 failure:
+---------
+```
+{: .no-copy }
+
+First you see warnings and deprecations.
+The `DEPRECATION` here is thrown by an MFTF dependency (Symfony) that is out of the scope for test writers and should be considered by MFTF contributors.
+In such cases, [report an issue][].
+
+Then, the MFTF reports that the test run took 52.43 seconds using 16 MB of the system RAM.
+And, finally, that there was `1 failure`.
+
+Next, the report provides you with details about the test fail.
+See interpretation after the following report.
+
+```terminal
+---------
+1) AdminMenuNavigationWithSecretKeysTestCest: Admin menu navigation with secret keys test
+ Test  tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:AdminMenuNavigationWithSecretKeysTest
+ Step  See "#something"
+ Fail  Failed asserting that  on page /admin/admin/system_config/index/key/678b7ba922c.../
+--> DASHBOARD
+SALES
+CATALOG
+CUSTOMERS
+MARKETING
+CONTENT
+REPORTS
+STORES
+SYSTEM
+FIND PARTNERS & EXTENSIONS
+Configuration
+admin
+1
+Store View: Default Config
+What is this?
+Save Config
+Country Options
+State Options
+Locale Options
+Store Information
+Store Name
+Store Phone Number
+Store Hours of Operation
+Countr
+[Content too long to display. See complete response in '/Users/dmytroshevtsov/Projects/vagrant/vagrant-magento/magento2ce/dev/tests/acceptance/tests/_output/' directory]
+--> contains "#something".
+
+Scenario Steps:
+
+ 23. $I->amOnPage("/admin/admin/auth/logout/") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:54
+ 22. // Cleaned cache types:
+config
+full_page
+ 21. $I->magentoCLI("cache:clean config full_page") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:52
+ 20. // Value was saved.
+ 19. $I->magentoCLI("config:set admin/security/use_form_key 0") at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:50
+ 18. $I->saveScreenshot() at tests/functional/Magento/FunctionalTest/_generated/default/AdminMenuNavigationWithSecretKeysTestCest.php:63
+```
+{: .no-copy }
+
+- `1) AdminMenuNavigationWithSecretKeysTestCest: Admin menu navigation with secret keys test`-the failed Codeception test is *AdminMenuNavigationWithSecretKeysTestCest*.It references to the PHP class that implemented the failed test.
```

Comment:
test` - the failed <-- add a space before and after -
*AdminMenuNavigationWithSecretKeysTestCest*. It references  <-- add a space between . and It